modules/browsemap/src/main/java/com/sygic/maps/module/browsemap/viewmodel/BrowseMapFragmentViewModel.kt

```diff
@@ -215,6 +221,10 @@ class BrowseMapFragmentViewModel internal constructor(
     }
 
     private fun getPoiDataAndNotifyObservers(viewObject: ViewObject<*>) {
+        if (detailsViewFactory == null) {
```

Comment:
nemalo by to kontrolovat aj `showDetailsView()`?

Reply:
👍 👍 👍